src/tests/utils/ApiTestSteps.ts

```diff
@@ -143,3 +164,129 @@ export async function getYotiSessionsInstructions(sessionId:any): Promise<any> {
 		return error.response;
 	}
 }
+
+export function generateRandomAlphanumeric(substringStart: number, substringEnd: number): string {
+	const result = Math.random().toString(36).substring(substringStart, substringEnd);
+	return result;
+}
+
+export async function getSessionById(sessionId: string, tableName: string): Promise<ISessionItem | undefined> {
+	const dynamoDB = createDynamoDbClient();
+
+	const getSessionCommand = new GetCommand({
+		TableName: tableName,
+		Key: {
+			sessionId,
+		},
+	});
+
+	let session;
+	try {
+		session = await dynamoDB.send(getSessionCommand);
+	} catch (e: any) {
+		console.error({ message: "getSessionById - failed executing get from dynamodb:", e });
+	}
+
+	return session.Item as ISessionItem;
+}
+
+export async function updateYotiSessionId(sessionId: string, yotiSessionId: any, updatedYotiSessionId: string): Promise<void> {
```

Comment:
does `yotiSessionId` need to be `any`?

Reply:
commented unused api test method for now